apps/extension/src/clients/index.ts

```diff
@@ -19,10 +19,10 @@ export const getPenumbraPort = (serviceTypeName: string) => {
     } as InitChannelClientMessage,
     [transferPort],
   );
-  return port;
+  return Promise.resolve(port);
```

Comment:
I don't think we need to force this to be an async function. We can just pass it as synchronous.